dask_distance/__init__.py

```diff
@@ -15,6 +15,38 @@
 from . import _utils
 
 
+#######################################
+#                                     #
+#  Numeric vector distance functions  #
+#                                     #
+#######################################
+
+
+@_utils._broadcast_uv_wrapper
+def braycurtis(u, v):
+    """
+    Finds the Bray-Curtis distance between two 1-D arrays.
+
+    .. math::
+
+       \\frac{ \sum_{i} \lvert u_{i} - v_{i} \\rvert }{ \sum_{i} \lvert u_{i} + v_{i} \\rvert }
```

Comment:
This line was too long. Wrapping it in PR ( https://github.com/jakirkham/dask-distance/pull/36 ).